UNFCCC_GHG_data/UNFCCC_reader/Guinea/read_GIN_BUR1_from_pdf.py

```diff
@@ -0,0 +1,753 @@
+        # They should also not contain negative values according to IPCC methodology:
+        # https://www.ipcc-nggip.iges.or.jp/public/2006gl/
+        # Therefore, the rows are deleted from the table.
+        row_to_delete = df_trend_entity.index[df_trend_entity[0] == "3.D - Autres"][0]
+        df_trend_entity = df_trend_entity.drop(index=row_to_delete)
+
+        row_to_delete = df_trend_entity.index[
+            df_trend_entity[0] == "3.D.1 - Produits ligneux récoltés"
+        ][0]
+        df_trend_entity = df_trend_entity.drop(index=row_to_delete)
+
+        row_to_delete = df_trend_entity.index[
+            df_trend_entity[0] == "3.D.2 - Autres (veuillez spécifier)"
+        ][0]
+        df_trend_entity = df_trend_entity.drop(index=row_to_delete)
+
+    else:
+        tables_inventory_original = camelot.read_pdf(
+            str(input_folder / pdf_file), pages=page, flavor="lattice", split_text=True
+        )
+        df_trend_entity = tables_inventory_original[0].df[3:]
+
+    print("Reading complete.")
+
+    # Add columns
+    # 'data' prefix is needed for pd.wide_to_long() later
+    columns_years = [
+        "data1990",
+        "data1995",
+        "data2000",
+        "data2005",
+        "data2010",
+        "data2015",
+        "data2018",
+        "data2019",
+    ]
+    df_trend_entity.columns = ["orig_cat_name"] + columns_years
+
+    df_trend_entity = df_trend_entity.copy()
+
+    # unit is always Gg
+    df_trend_entity.loc[:, "unit"] = "Gg"
+
+    # only one entity per table
+    df_trend_entity.loc[:, "entity"] = entity
+
+    df_trend_entity.loc[:, "category"] = df_trend_entity["orig_cat_name"]
+
+    # Delete empty line for pages 132-137.
+    if page != "131":
+        row_to_delete = df_trend_entity.index[df_trend_entity["category"] == ""][0]
+        df_trend_entity = df_trend_entity.drop(index=row_to_delete)
+
+    df_trend_entity["category"] = df_trend_entity["category"].replace(
+        inv_conf["cat_codes_manual"]["trend"]
+    )
+
+    df_trend_entity.loc[:, "category"] = df_trend_entity["category"].str.replace(
+        ".", ""
+    )
+    df_trend_entity.loc[:, "category"] = df_trend_entity["category"].str.replace(
+        "\n", ""
+    )
+
+    def repl(m):
+        return m.group("code")
+
+    df_trend_entity.loc[:, "category"] = df_trend_entity["category"].str.replace(
+        inv_conf["cat_code_regexp"], repl, regex=True
+    )
+
+    df_trend_entity = df_trend_entity.reset_index(drop=True)
+
+    print("Created category codes.")
+
+    for year in columns_years:
+        df_trend_entity.loc[:, year] = df_trend_entity[year].str.replace(",", ".")
+        df_trend_entity.loc[:, year] = df_trend_entity[year].str.replace("NE1", "NE")
+
+    # make sure all col headers are str
+    df_trend_entity.columns = df_trend_entity.columns.map(str)
+
+    df_trend_entity = df_trend_entity.drop(columns=["orig_cat_name"])
+
+    # TODO wide in IF gibt es convert_wide_dataframe_if
```

Comment:
`convert_long_dataframe_if` internally uses `pd.long_to_wide`so you're actually going back and forth here. Leave it as is here for time reasons, but keep in mind for the future